simulatingrisk/risky_food/README.md

```diff
@@ -0,0 +1,32 @@
+# Risky Food Simulation
+
+## Summary
+
+Game: risky food source is 3 if **N**, 1 if **C**; safe source is 2
+
+- **N**: non-contaminated 
+- **C**: contaminated
+
+Every agent gets a parameter `r` between 0 and 1.  [or DISCRETE: 8 buckets etc.]
+
+EACH ROUND:
+- Nature selects a probability `p` for **N**
+- For each agent: if `r` > `p`, then they choose RISKY; else SAFE
+- Nature flips a coin with bias `p` for **N**, and announces **N** or **C**
+- If **N**: everyone who chose RISKY gets 3, everyone who chose SAFE gets 2
+- If **C**: everyone who chose RISKY gets 1, everyone SAFE 2
+- Reproduce in proportion to payoff
+  - Either agent gets # of offspring = payoff [they replace–original “dies off”]
+  - OR: take the total payoff for RISKYs over total for everyone, there are that proportion of RISKYs in the new population
+
+END ROUND
```

Comment:
I like this semi-formal way of describing a 'round'! Very clear articulation of the steps/rules of the game.

Reply:
this is all from Lara! I just added some formatting. very clear, right?